docs/README.md

```diff
@@ -76,3 +76,11 @@ agent_list = [
     agents.PlayerAgent(agent_control="wasd"), # W,A,S,D to move, E to lay bomb
 ]
 ```
+
+## Submitting an Agent.
+
+In order to submit an agent, you need to create an account at [pommerman.com](https://pommerman.com). This will link pommerman to your Github account. 
```

Comment:
we don't actually need people to link their github account. the approach using deploy keys gets around that.